coalib/output/ConsoleInteraction.py

```diff
@@ -855,38 +855,58 @@ def show_enumeration(console_printer,
     console_printer.print()
 
 
+def check_deprecation(show_description,
+                      show_params):
+    """
+    Shows a deprecation warning message if the parameters
+    show_description or show_params are not None.
+
+    :param show_description: This parameter is being checked
+                             for deprecation.
+    :param show_params:      This parameter is being checked
+                             for deprecation.
+    """
+    param_dict = {'show_description': show_description,
```

Comment:
if you must have a dict here, this dict will grow over time.

Put the entries on separate lines, following by a trailing `,` , so additional entries are easy to add.